src/views/ops/Adhoc/QuickJob.vue

```diff
@@ -552,12 +556,12 @@ export default {
       })
     },
     setBtn() {
-      if (this.executionInfo.status.value !== 'running') {
+      if (!this.isRunning) {
         clearInterval(this.executionInfo.cancel)
         this.toolbar.left.run.icon = 'fa fa-play'
       }
-      this.toolbar.left.run.isVisible = this.executionInfo.status.value === 'running'
-      this.toolbar.left.stop.isVisible = this.executionInfo.status.value !== 'running'
+      this.toolbar.left.run.isVisible = this.isRunning
+      this.toolbar.left.stop.isVisible = !this.isRunning
     },
     onSubmitVariable(parameters) {
       this.parameters = parameters
```

Comment:
This code seems generally fine, no obvious errors or inefficiencies found at the moment. However, here's some general advice on possible optimizations:

- Use object destructuring to simplify access of `classification` attribute within the component method.
- If you're using dynamic properties like `_t`, make sure it resolves with correct types before accessing them (`typeof`) since TypeScript does not resolve type inference for unknown objects.
- Ensure consistent spacing around operators, especially where they align horizontally.

Keep in mind, these improvements would need specific context based on the environment and application being run.